ICSharpCode.Decompiler/TypeSystem/VarArgInstanceMethod.cs

```diff
@@ -129,6 +129,8 @@ public IMethod Specialize(TypeParameterSubstitution substitution)
 			get { return baseMethod.IsExtensionMethod; }
 		}
 
+		bool IMethod.IsLocalFunction => false;
```

Comment:
I'd just forward this to the base implementation; in principle I don't see why it should be impossible to construct a VarArgInstanceMethod for a local function.

Reply:
The compiler explicitly prohibits this, but that might be changed in the future. Fixed.